x-pack/plugins/enterprise_search/server/lib/enterprise_search_request_handler.test.ts

```diff
@@ -408,7 +408,7 @@ describe('EnterpriseSearchRequestHandler', () => {
     it('sets the value of wsOAuthTokenPackage in a cookie', async () => {
       const mockNow = 'Thu, 04 Mar 2021 22:40:32 GMT';
       const mockInAnHour = 'Thu, 04 Mar 2021 23:40:32 GMT';
-      const tokenPackage = 'sEkReTZ';
+      const tokenPackage = 'some_encrypted_secrets';
```

Comment:
Just curious & for my own edification - do we ever anticipate forwarding secrets for more use cases than just oAuth in the future?

Reply:
I don't see this token package containing anything that isn't used for connecting a source but that doesn't strictly mean it is data related to OAuth, if that makes sense.

Reply:
Definitely, very good to know!